# This message will contain information about the lanes in a road sections
geometry_msgs/Point32[] left_line
geometry_msgs/Point32[] middle_line
geometry_msgs/Point32[] right_line
